plentyofpets/lib/components/admin_homepage_buttons

```diff
@@ -0,0 +1,134 @@
+import 'package:flutter/material.dart';
+import 'package:admin_homepage_navigation/add_pet.dart';
+
+class AddPetCardButton extends StatelessWidget {
+  final String buttonText;
+
+  AddPetCardButton({required this.buttonText});
+
+  @override
+  Widget build(BuildContext context) {
+    return SizedBox(
+      width: 150,
```

Comment:
I like how you've pulled these buttons out into classes. Is this sized box like the border of the button or some background? I'm just worried that these fixed height/width values might not always work well depending on the text that is passed in. Maybe it's possible to use Padding and achieve a similar affect? Something like

`Padding(
    padding: EdgeInsets.all(15),
    child: Card(...)
)`

Reply:
ah good idea, I'll try to put it inside a Padding to avoid uneven shaped boxes